sys-boot/grub/grub-9999.ebuild

```diff
@@ -119,6 +122,19 @@ QA_PRESTRIPPED="
 "
 
 pkg_pretend() {
+	if [[ ${GRUB_PLATFORMS} =~ "arm64" ]] ; then
+		ebegin "Checking for aarch64-cros-linux-gnu-gcc"
+		if type -p aarch64-cros-linux-gnu-gcc > /dev/null ; then
```

Comment:
Perhaps it would be better to add `grub_platforms_arm64? ( cross-x86_64-cros-linux-gnu/gcc )` to DEPEND rather than use a run-time check.